pkg/cosign/payload.go

```diff
@@ -20,20 +20,22 @@ import (
 	v1 "github.com/google/go-containerregistry/pkg/v1"
 )
 
-func Payload(img v1.Descriptor, a map[string]string) ([]byte, error) {
+type ContainerImagePayload struct {
```

Comment:
Maybe drop Container? We call it Image below.

Reply:
done